services/metadata_service/api/admin.py

```diff
@@ -96,11 +96,11 @@ async def healthcheck(self, request):
     async def get_authorization_token(self, request):
         """
         ---
-        description: this is used exclusively for sandbox auth
+        description: This endpoint is used exclusively for sandbox auth
         tags:
         - Auth
         produces:
-        - text/plain
+        - application/json
```

Comment:
Have you verified that this change wouldn't affect the sandbox? We have a similar issue where the `heartbeat` endpoint returns an actual JSON but is tagged as `text/plain` which has caused issues in the metaflow codebase before.

Reply:
I am going to rework things and push a PR to make it so that everything works on application/json. I will also check with Ferras if there was a reason for the string only implementation.